### PR TITLE
[GLUTEN-2019][CH]Bug fix not allow quotes in txt format table read

### PR DESCRIPTION
**Pull request by KevinyhZou**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

(Fixes: \#2019)

## How was this patch tested?

unit tests

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

File: cpp-ch/local-engine/Storages/SubstraitSource/TextFormatFile.cpp [hunk 1]
```diff
@@ -29,6 +29,8 @@ FormatFile::InputFormatPtr TextFormatFile::createInputFormat(const DB::Block & h
         format_settings.hive_text.input_field_names.push_back(name);
     format_settings.hive_text.fields_delimiter = file_info.text().field_delimiter()[0];
     format_settings.csv.empty_as_default = false;
+    format_settings.csv.allow_single_quotes = false;
+    format_settings.csv.allow_double_quotes = false;
```
Comment (loneylee) [inline cpp-ch/local-engine/Storages/SubstraitSource/TextFormatFile.cpp:33]: hive里面允许用`"`,`'`作为quota，这里如果写死，是不是当配置了quota等于`"`就会出问题

Comment (KevinyhZou) [reply cpp-ch/local-engine/Storages/SubstraitSource/TextFormatFile.cpp:33]: OK, 我把这个地方做成可配置化的

Comment (KevinyhZou) [reply cpp-ch/local-engine/Storages/SubstraitSource/TextFormatFile.cpp]: Hello, 我这里参考`ExcelTextFormatFile.cpp` 中的作业，考虑了hive中使用`'`, `"` 作为quote 的场景，麻烦再review下
@loneylee

**Comment by loneylee** [comment]
LGTM

**Pull request merged**